client/scripts/views/modals/ragequit_modal.ts

```diff
@@ -1,60 +0,0 @@
-import 'modals/ragequit_modal.scss';
```

Comment:
we should add this file back, in the future we'll wanna support moloch and this will be a good reference for a future iteration of this modal